src/instructlab/model/chat.py

```diff
@@ -59,8 +59,8 @@
 """
 
 CONTEXTS = {
-    "default": get_sysprompt(),
-    "cli_helper": "You are an expert for command line interface and know all common commands. Answer the command to execute as it without any explanation.",
+    "default": get_sysprompt,
+    "cli_helper": get_cli_helper_sysprompt,
```

Comment:
are these functions? will they be called?

Reply:
yes they are functions
they are called at various places in this PR